roslib/Header header
int32 tick
string name
TimelineDescription[] internal
TimelineDescription[] external 
TimelineDescription[] actions
